ibis/backends/mysql/registry.py

```diff
@@ -184,3 +184,17 @@ def _json_get_item(t, op):
         ops.JSONGetItem: _json_get_item,
     }
 )
+
+_invalid_operations = {
+    ops.CumulativeAll,
+    ops.CumulativeAny,
+    ops.CumulativeMax,
+    ops.CumulativeMean,
+    ops.CumulativeMin,
+    ops.CumulativeSum,
+    ops.NTile,
```

Comment:
Should we also remove geo operation? It is not covered by any test.
```
    ops.CumulativeAll,
    ops.CumulativeAny,
    ops.CumulativeMax,
    ops.CumulativeMean,
    ops.CumulativeMin,
    ops.CumulativeSum,
    ops.NTile,
    ops.GeoArea,
    ops.GeoAsBinary,
    ops.GeoAsEWKB,
    ops.GeoAsEWKT,
    ops.GeoAsText,
    ops.GeoAzimuth,
    ops.GeoBuffer,
    ops.GeoCentroid,
    ops.GeoContains,
    ops.GeoContainsProperly,
    ops.GeoCoveredBy,
    ops.GeoCovers,
    ops.GeoCrosses,
    ops.GeoDFullyWithin,
    ops.GeoDWithin,
    ops.GeoDifference,
    ops.GeoDisjoint,
    ops.GeoDistance,
    ops.GeoEndPoint,
    ops.GeoEnvelope,
    ops.GeoEquals,
    ops.GeoGeometryN,
    ops.GeoGeometryType,
    ops.GeoIntersection,
    ops.GeoIntersects,
    ops.GeoIsValid,
    ops.GeoLength,
    ops.GeoLineLocatePoint,
    ops.GeoLineMerge,
    ops.GeoLineSubstring,
    ops.GeoNPoints,
    ops.GeoOrderingEquals,
    ops.GeoOverlaps,
    ops.GeoPerimeter,
    ops.GeoSRID,
    ops.GeoSetSRID,
    ops.GeoSimplify,
    ops.GeoStartPoint,
    ops.GeoTouches,
    ops.GeoTransform,
    ops.GeoUnaryUnion,
    ops.GeoUnion,
    ops.GeoWithin,
    ops.GeoX,
    ops.GeoY
```

Reply:
Yeah, let's remove geospatial ops here.

Reply:
This can be done in a follow-up so we can merge this one now :)